### ISSUE TITLE
Build is Failing because of a jar-dependencies with 5.6

### ISSUE THREAD

**Issue by original-brownbear** [logstash-plugins/logstash-input-kafka#241]
https://travis-ci.org/logstash-plugins/logstash-input-kafka/jobs/304965466#L1081

```sh
undefined method `vendor_jars!' for #<Jars::Installer:0x5f9edf14>
/home/travis/build/logstash-plugins/logstash-input-kafka/Rakefile:14:in `(root)'
org/jruby/RubyProc.java:281:in `call'
org/jruby/RubyArray.java:1613:in `each'
org/jruby/RubyArray.java:1613:in `each'
org/jruby/RubyArray.java:1613:in `each'
/home/travis/.rvm/gems/jruby-1.7.27/gems/rake-12.2.1/exe/rake:27:in `(root)'
org/jruby/RubyKernel.java:1059:in `load'
org/jruby/RubyKernel.java:1079:in `eval'
/home/travis/.rvm/gems/jruby-1.7.27/bin/jruby_executable_hooks:15:in `(root)'
Tasks: TOP => vendor => install_jars
(See full trace by running task with --trace)
```

seems to me, we're not properly loading the `jar-dependencies` gem somehow and using the built in version of `JRuby` which doesn't have the method.

**Comment by original-brownbear** [comment]
@jakelandis how close are we to moving this stuff (plugins) to Jenkins? :D 

This is some issue with the Travis ENV that I already wasted an hour trying to track down. The exact same LS + JRuby versions work clean for me locally. Is it worth sinking a lot of time into this?

**Comment by jakelandis** [comment]
@original-brownbear - AFAIK there are no plans to move the plugin builds to Jenkins.

**Comment by original-brownbear** [comment]
@jakelandis ok thanks :) then we have to find a solution here ...

**Comment by robbavey** [comment]
Fixed by #263
